src/components/store/HelpModal/index.tsx

```diff
@@ -0,0 +1,101 @@
+import { Title } from '@/components/common';
+import Step1 from '@/public/assets/graphics/store/step1.svg';
+import Step2 from '@/public/assets/graphics/store/step2.svg';
+import Step3 from '@/public/assets/graphics/store/step3.svg';
+import Step4 from '@/public/assets/graphics/store/step4.svg';
+import ArrowLeftIcon from '@/public/assets/icons/arrow-left.svg';
+import ArrowRightIcon from '@/public/assets/icons/arrow-right.svg';
+import CloseIcon from '@/public/assets/icons/close-icon.svg';
+import { ReactNode, useEffect, useRef } from 'react';
+import styles from './style.module.scss';
+
+interface StepProps {
+  step: number;
+  children?: ReactNode;
+}
+const Step = ({ step, children }: StepProps) => {
+  const Left = step > 1 ? 'a' : 'span';
+  const Right = step < 4 ? 'a' : 'span';
+  return (
+    <div className={styles.step} id={`step-${step}`}>
+      {children}
+      <div className={styles.stepControls}>
+        <Left className={styles.stepControl} href={`#step-${step - 1}`}>
+          <ArrowLeftIcon aria-label="Previous page" />
+        </Left>
+        <Right className={styles.stepControl} href={`#step-${step + 1}`}>
+          <ArrowRightIcon aria-label="Next page" />
+        </Right>
+      </div>
+    </div>
+  );
+};
+
+interface HelpModalProps {
+  open: boolean;
+  onClose: () => void;
+}
+
+const HelpModal = ({ open, onClose }: HelpModalProps) => {
+  const ref = useRef<HTMLDialogElement>(null);
+
+  useEffect(() => {
+    ref.current?.addEventListener('click', e => {
+      if (e.target === e.currentTarget) {
+        onClose();
+      }
+    });
+  }, [onClose]);
```

Comment:
If I'm not mistaken, this isn't necessary anymore since I decided to just disable ESLint